### ISSUE TITLE
KITTI and SceneFlow pretrained model

### ISSUE THREAD

**Issue by YJLCV** [antabangun/coex#11]
Hi,
Thanks for your great work. I'm trying to run your code. And I found that you gave a model(logs/stereo/CoEx/version_0/checkpoints/last.ckpt), but I can't reproduce the results of the paper when I tested this model. Is this model from sceneflow or kitti, and whether the pre-training models of both datasets can be opened?

Thank you very much!

**Comment by antabangun** [comment]
Hi,
Thanks for your interest!
I believe the uploaded checkpoint was the KITTI checkpoint, IIRC, which we used to get the KITTI results on the paper. Unfortunately, I haven't worked on this recently, so I don't have the checkpoint for both datasets. But I believe you can reproduce the training using the uploaded codes with pretty close accuracy to what we reported.
A pointer I can add for the training is the current results on the KITTI leaderboard were obtained after adding scaling and stretching augmentation with longer training, and it gives improved results compared to the paper.
